PivotCalibration/Logic/vtkSlicerPivotCalibrationLogic.h

```diff
@@ -69,10 +69,14 @@ class VTK_SLICER_PIVOTCALIBRATION_MODULE_LOGIC_EXPORT vtkSlicerPivotCalibrationL
   // Returns with false on failure
   bool ComputeSpinCalibration( bool snapRotation = false ); //Note: The neede orientation protocol assumes that the shaft of the tool lies along the positive x-axis
 
+  // Flip the direction of the shaft axis
+  void FlipShaftDirection();
```

Comment:
Could you add a few lines (probably in the general description of the class) that explains the link between the spin and pivot calibration and shaft direction and also if there is any difference in what order the two calibrations are performed?